cpp/include/cudf/legacy/io_types.hpp

```diff
@@ -38,7 +44,16 @@ struct source_info {
   std::string filepath;
   std::pair<const char*, size_t> buffer;
   std::shared_ptr<arrow::io::RandomAccessFile> file;
-
+  std::string topicname;
```

Comment:
where is this member used?

Reply:
It is not. I had to do some heavy merging and looks like I simply overlooked removing it. Thanks for pointing it out.